packages/shared-state-network_nodes/files/usr/lib/lua/network-nodes.lua

```diff
@@ -0,0 +1,100 @@
+local shared_state = require("shared-state")
+local utils = require("lime.utils")
+local config = require("lime.config")
+
+local network_nodes = {}
+
+function network_nodes.node(hostname, member, fw_version, board, ipv4, ipv6)
```

Comment:
this function feels like a private function within the module... could it be the case?

Reply:
Thanks for pointing it. I've pushed a refactor for this and all the non public functions to be accesed with a leading underscore (as they need to be reachable for the tests).